### PR TITLE
Fix non-ignored test, check all tests are executed

### PR DESCRIPTION
**Pull request by MarcoGorelli**
In `.github/workflows/pytest.yml` there was `--ignore=pymc3/tests/test_shape_handling` instead of `--ignore=pymc3/tests/test_shape_handling.py` :flushed: . So, fixed + pre-commit check to ensure all tests are executed added

### CODE REVIEW

File: .pre-commit-config.yaml
Comment (MarcoGorelli) [inline .pre-commit-config.yaml:46]: this is the only new hook here, in the others I just sorted the keys

File: scripts/check_all_tests_are_covered.py
Comment (MarcoGorelli) [inline scripts/check_all_tests_are_covered.py:20]: note: these are sets, so `<=` means "is a subset of"

**Comment by MarcoGorelli** [comment]
32m 8s  ... _nearly_ broke half-hour, GHA is really impressive

File: .pre-commit-config.yaml
Comment (AlexAndorra) [reply .pre-commit-config.yaml:46]: Good call adding this hook 👍

**Review by AlexAndorra — APPROVED** [review]
Good catch @MarcoGorelli! Thanks for the comments in the PR, makes it easier to review 👍 
It seems that Travis is still doing its thing though -- I thought @twiecki had deactivated it 🤔

**Comment by twiecki** [comment]
I think I disabled it, not sure why it still shows up...

**Comment by Spaak** [comment]
In #4260 all GHA tests are passing, but status is still "waiting on Travis" which prevents a merge. Is it possible that the reason is that #4260 was filed *before* this one was merged? Any way to force-delete the Travis check?

**Comment by twiecki** [comment]
No that has nothing to do with it, somehow travis is still installed but but non-functional but I can't find where.

**Pull request merged**
